### ISSUE TITLE
Scroll breaks when attaching an image to a custom content type

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#572]
```
Please make sure you are including enough information to make this a valid
bug report: https://code.google.com/p/wordpress-custom-content-type-
manager/wiki/ReportingBugs

What steps will reproduce the problem? (Be *specific*)
1. Install plugin
2. Create custom type
3. Add a custom field to the type that uses images
4. Add new post to the type (notice how you can scroll the page to get to 
custom fields)
5. Add an image (or at least bring up the menu to select an image then close it 
out)
6. Observe how scrolling is now broken on the page. You cannot get to the top 
to hit save unless you tab around... 

What is the expected output?
Image would be added and the page would be able to scroll to the top so you can 
hit save.

What do you see instead?
The page, but it has no scrolling.


Does the problem continue if you disable all other plugins? (plugin
conflicts cause a LOT of problems, so please isolate the problem)
Yes, this is a vanilla install of WP with only this plugin.

Attached is a video showing the bug on a vanilla install of WP4.0


*SYSTEM INFO* 
------------------------ 
Plugin Version: 0.9.7.13-pl
WordPress Version: 4.0
PHP Version: 5.4.29
MySQL Version: 5.5.38
Server OS: Linux
Language: 
------------------------ 
ACTIVE PLUGINS: 
 * Custom Content Type Manager : Advanced Custom Post Types v.0.9.7.13 [http://code.google.com/p/wordpress-custom-content-type-manager/]
------------------------ 
CURRENT THEME: Twenty Fourteen v.1.2 http://wordpress.org/themes/twentyfourteen
```

Original issue reported on code.google.com by `olympusa...@gmail.com` on 18 Sep 2014 at 3:02

Attachments:
- [CCT_bug.swf](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-572/comment-0/CCT_bug.swf)

**Comment by GoogleCodeExporter** [comment]
```
This has been reported on WordPress.org too. Also a temp fix is listed there. 
https://wordpress.org/support/topic/image-custom-field-bug-after-wp-40?replies=3
#post-6026596
```

Original comment by `olympusa...@gmail.com` on 18 Sep 2014 at 3:14

**Comment by GoogleCodeExporter** [comment]
```
I have this problem too

```

Original comment by `simonjpa...@gmail.com` on 25 Sep 2014 at 4:33

**Comment by GoogleCodeExporter** [comment]
```
I have problem with this too. Please for fast resolution. Temp fix didn't work 
for me :/
```

Original comment by `pmare...@gmail.com` on 1 Oct 2014 at 1:02

**Comment by GoogleCodeExporter** [comment]
```
Refer to my answer in this other thread
http://code.google.com/p/wordpress-custom-content-type-manager/issues/detail?id=
573#c1
```

Original comment by `guidonav...@gmail.com` on 6 Oct 2014 at 6:25

**Comment by GoogleCodeExporter** [comment]
```
The temp fix stated here 
https://wordpress.org/support/topic/image-custom-field-bug-after-wp-40?replies=3
#post-6026596 might need an !important to work. So put:

body.modal-open {
     overflow: visible !important;
}

into your wp-content/plugins/custom-content-type-manager/css/manager.css and 
you should be fine. Not elegant, but works!



```

Original comment by `st.ploes...@gmail.com` on 30 Oct 2014 at 11:34

**Comment by GoogleCodeExporter** [comment]
```
add 

jQuery("body").removeClass("modal-open"); 

to

wp-content/plugins/custom-content-type-manager/js/thickbox.js

after

jQuery("#TB_load").remove();

you're fine. 
```

Original comment by `volker.f...@gmail.com` on 8 Dec 2014 at 1:10

**Comment by GoogleCodeExporter** [comment]
```
Fixed applied on the core files.
```

Original comment by `daniel.e...@gmail.com` on 17 Dec 2014 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
I attached here the updated file which has the fix for this issue.
```

Original comment by `daniel.e...@gmail.com` on 19 Dec 2014 at 12:09

Attachments:
- [thickbox.js](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-572/comment-8/thickbox.js)

**Comment by GoogleCodeExporter** [comment]
```
I only needed the one change, from comment #8.  And it works!  thanks.
```

Original comment by `jonda...@gmail.com` on 13 Apr 2015 at 2:12

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.9.8
```

Original comment by `ever...@fireproofsocks.com` on 6 May 2015 at 11:52
- Changed state: **Fixed**
